modules/web/src/app/dynamic/enterprise/metering/schedule-config/add-dialog/component.ts

```diff
@@ -182,6 +185,7 @@ export class MeteringScheduleAddDialog implements OnInit, OnDestroy {
       schedule: this._schedule,
       interval: this._interval,
       retention: this._retention,
+      types: this.form.get(Controls.Types).value,
```

Comment:
so all types of reports will be sending even if it's not custom Configuration ?

Reply:
By default all types are selected when a schedule is created but I think here it doesn't make sense to send `types`  in payload when custom configuration option is not selected. I'll update the payload.